zubhub_frontend/zubhub/src/components/breadCrumb/breadCrumb.jsx

```diff
@@ -61,8 +60,8 @@ function BreadCrumb() {
   };
 
   const createPathList = () => {
-    let pathList = match.path.split('/');
-    let pathListUrl = match.url.split('/');
+    let pathList = window.location.pathname.split('/');
+    let pathListUrl = window.location.href.split('/');
```

Comment:
why are we using `window` instead of `match`?

Reply:
@coderatomy

Reply:
no match object in rr v6